packages/blade/src/components/Tabs/tabTokens.ts

```diff
@@ -0,0 +1,174 @@
+import type { TabsProps } from './types';
+import type { DotNotationSpacingStringToken } from '~utils/types';
+
+type Devices = 'desktop' | 'mobile';
+type TabSizes = NonNullable<TabsProps['size']>;
+type TabVariants = Exclude<NonNullable<TabsProps['variant']>, 'borderless'>;
+type TabOrientation = NonNullable<TabsProps['orientation']>;
+
+type TabItemPadding = Record<
+  TabVariants,
+  Record<TabOrientation, Record<Devices, Record<TabSizes, DotNotationSpacingStringToken>>>
+>;
+
+const paddingY: TabItemPadding = {
```

Comment:
does it seriously needs this complex padding so many permutations and combinations?

Reply:
:p you are looking at the simplified version, previously it was [monstrous](https://github.com/razorpay/blade/pull/1694/commits/9a5d240a88c9422514a2d67fa131857c0e93cff8#diff-18f9bb44412c8a625a3f751777eb97fd77db23be76807564f80c6bd07fdd63f4R25-R188). 

This approach is a whole lot easier to maintain and update later than using ternaries to switch between different variants.

Reply:
@coderabbit